pkg/config/config.go

```diff
@@ -268,6 +270,10 @@ func load(fs *flag.FlagSet, args []string, loader func(string, bool, *Config) er
 		os.Exit(0)
 	}
 
+	if cfg.ExperimentalConfigURLs {
+		loader = LoadRemote
+	}
```

Comment:
I think this would be unexpected from a testing point of view, see suggestion for another way of handling it